terminus-ui/csv-entry/src/csv-entry.component.scss

```diff
@@ -40,6 +40,7 @@
   .c-csv-entry__scroll {
     // Odd height used to ensure that the table cuts off mid-cell so the user knows there is more content
     max-height: 28.5em;
+    overflow-x: -moz-hidden-unscrollable;
```

Comment:
We should probably add a note about why this is needed. Since we don't officially support anything other than chrome I could see this getting deleted down the line.

Reply:
Ditto... a comment would definitely be nice. Was adding it but probably got removed when doing clean up (tried bunch of different things). Although I’d say this name is very Firefox specific :)